plat/mediatek/mt6795/include/mcucfg.h

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
This is very simlilar to plat/mediatek/mt8173/include/mcucfg.h. Is any code sharing possible?